src/target_root.c

```diff
@@ -209,7 +209,10 @@ static enum swupd_code stage_single_file(struct file *file, struct manifest *mom
 		debug("Target directory does not exist: %s\n", target_path);
 		if (mom) {
 			debug("Attempting to fix missing directory\n");
-			verify_fix_path(dir, mom);
+			if (!verify_fix_path(dir, mom)) {
```

Comment:
if you are checking the success of creating the directory here then we don't need to check again in line 225

Reply:
you missed this comment

Reply:
ahh maybe not, looks like my review hadn't refreshed yet